components/Slider/README.md

```diff
@@ -215,3 +220,34 @@ id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility {a11y}
+
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility {a11y}
+
+To help ensure your slider is accessible to as many users as possible, please be sure to review the following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate `accessibilityLabel` value for your slider. This should reflect what the slider affects.
+
+#### Objective - C
+```objc
+slider.accessibilityLabel = @"Volume Slider";
+``` 
+
+#### Swift
+```swift
+slider.accessibilityLabel = "Volume Slider"
+```
+
+### `-accessibilityActivate`
+ 
+ Our implementation closely resembles what UISlider does but it's not an exact match. On an `accessibilityActivate` we move one sixth of the amount between the current value and the midpoint value.
```

Comment:
In general, are we documenting slight differences like this? Do we any rules of thumb for when to document a feature and when not to? 

I feel like this section should be devoted to what a client *must* review to make components more accessible.

Reply:
It's not an exact match, @jverkoey or @randallli what do you guys think? Should we remove this section?